be/src/exprs/expr.cpp

```diff
@@ -292,6 +294,8 @@ Status Expr::create_vectorized_expr(starrocks::ObjectPool* pool, const starrocks
         } else if (texpr_node.fn.name.function_name == "is_null_pred" ||
                    texpr_node.fn.name.function_name == "is_not_null_pred") {
             *expr = pool->add(vectorized::VectorizedIsNullPredicateFactory::from_thrift(texpr_node));
+        } else if (texpr_node.fn.name.function_name == "array_map") {
+            *expr = pool->add(new vectorized::ArrayMapExpr(texpr_node));
```

Comment:
Do need to handle each array high order functions?

Reply:
not really, for example, array_filter(lambda func, arr1...) = array_filter(arr1, array_map(lambda func, arr1...), then array_filter just is a scalar func to implement array_filter(array, bool_array).